### ISSUE TITLE
Issues building from quickstart instructions

### ISSUE THREAD

**Issue by Ragnyll** [fermyon/bartholomew#121]
I'm trying to build the `bart` binary from source but the prescribed make command fails, and it's unclear what is going wrong that I need to fix.

The [docs](https://github.com/fermyon/bartholomew/blob/118b074215629e1339d0f7812abd962ba09cfb83/docs/content/quickstart.md) state the following steps for building:
```zsh
% git clone https://github.com/fermyon/bartholomew.git
% cd bartholomew
% make bartholomew
```

which results in 
```
make: *** No rule to make target 'bartholomew'.  Stop.
```

I don't think `make bartholomew` is the right make command based off this previous [issue comment https://github.com/fermyon/bartholomew/issues/61#issuecomment-1092467847] by @radu-matei . I'm pretty sure it should be make bart. However the results there are still odd.

```zsh
% make bart
cargo build --release --manifest-path=bart/Cargo.toml
    Finished release [optimized] target(s) in 0.55s
```

Calling `bart` obviously results in the following:

```zsh
zsh: command not found: bart
```

It strikes me as odd that the cargo commands from make bart build so quickly. Also, is the build target supposed to be a bin or a wasi file? I'd think it would be a bin since i need to execute that, but i cant find the bin in the target directory after calling make.

In case it helps with debugging here's a little information on the host I'm running this from
```
Linux version 5.19.2-arch1-2 (linux@archlinux) (gcc (GCC) 12.2.0, GNU ld (GNU Binutils) 2.39.0) #1 SMP PREEMPT_DYNAMIC Sat, 20 Aug 2022 16:11:19 +0000
```

And I do have spin installed from main.
```
% spin --version
spin 0.5.0 (c3bbfeb 2022-09-02)
```

**Comment by radu-matei** [comment]
`make bart` generates a binary you can find in `target/release/bart` — does that help?

**Comment by Ragnyll** [comment]
yeah, that at least gets me the binary i need to install.

should the make command drop it in `/usr/bin/` though? because in order to make the template site it will need to be callable from other directories (which can be done through relative pathing, but i'm not sure thats something that should be implicit from the build instructions)

**Comment by Ragnyll** [comment]
I opened a minor pr to make the `make bart` lifecycle clear in the docs. If y'all want this feel free to review and allow the CI to run. 

https://github.com/fermyon/bartholomew/pull/122

**Comment by Ragnyll** [comment]
The merged pr #122 resolves this.
